src/data/fintype/card_embedding.lean

```diff
@@ -0,0 +1,169 @@
+/-
+Copyright (c) 2021 Eric Rodriguez. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Rodriguez
+-/
+import data.fintype.card
+import data.nat.factorial
+
+/-!
+# Birthday Problem
+
+This file establishes the cardinality of `α ↪ β` in full generality.
+-/
+
+open_locale classical
```

Comment:
I think you should try to remove this - it's going to cause problems with the computable instance I propose. You can replace it with:

* The definition in #7383
* Some `[decidable_eq β]` arguments below
* This lemma:
```
instance {α β} [fintype α] [fintype β] [decidable_eq α] [decidable_eq β] : decidable_eq (α ↪ β) :=
λ a b, decidable_of_iff (⇑a = b) function.embedding.coe_injective.eq_iff
```

Reply:
fingers crossed the last commit solved this, but can't check until CI wants to :(